code/modules/projectiles/guns/ballistic.dm

```diff
@@ -676,7 +676,7 @@ GLOBAL_LIST_INIT(gun_saw_types, typecacheof(list(
 
 /obj/item/suppressor
 	name = "suppressor"
-	desc = "A syndicate small-arms suppressor for maximum espionage."
+	desc = "A Scarborough Arms small-arms suppressor for maximum espionage." //SKYRAT EDIT
```

Comment:
couldn't this be modualirized by making a suppressor subtype with the new description and having the gun ordering system use that one? ((It's perfectly fair if you don't want to go through that, this is a very minor change.))